src/scene.cpp

```diff
@@ -27,9 +27,9 @@
 #include "transition.h"
 
 #ifndef NDEBUG
-#define DEBUG_VALIDATE() Scene::DebugValidate(__PRETTY_FUNCTION__)
```

Comment:
Could also use [__FUNCDNAME__](https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=vs-2017) or such for msvc

Reply:
yeah, was not sure if I want to add another preprocessor if-block or simply hardcode the function names.

Reply:
The scope of this is so limited it's probably fine to just hard code the names instead of doing nasty stuff with macros.